main.tf

```diff
@@ -8,6 +8,24 @@ locals {
   vault_name       = coalesce(var.vault_name, module.this.id)
   vault_id         = join("", local.vault_enabled ? aws_backup_vault.default.*.id : data.aws_backup_vault.existing.*.id)
   vault_arn        = join("", local.vault_enabled ? aws_backup_vault.default.*.arn : data.aws_backup_vault.existing.*.arn)
+  compat_rule = {
```

Comment:
what does `compat` mean ? Is this a typo for `compact` ?

Reply:
Or is it short for compatible ?

Reply:
Yes, "compat" is for "compatibility" (I've seen this usage in https://cgit.freebsd.org/src/tree/sys/compat/linux). I wanted to make it apparent that this record is for backwards compatibility and is likely not staying in the code forever. Let me know if you'd like me to change it.